src/ucs/sys/topo/base/topo.c

```diff
@@ -207,34 +207,53 @@ ucs_topo_bus_id_to_sysfs_path(const ucs_sys_bus_id_t *bus_id, char *path,
                               size_t max)
 {
     const size_t prefix_length = strlen(UCS_TOPO_SYSFS_PCI_PREFIX);
-    char link_path[PATH_MAX];
+    ucs_status_t status        = UCS_OK;
+    char *link_path            = NULL;
```

Comment:
no need to initialize link_path and status

Reply:
done